### PR TITLE
Add rest/command_line/filter/statistics to the list of reloadables

### PR DESCRIPTION
**Pull request by bdraco**
Backend: https://github.com/home-assistant/core/pull/39262 and https://github.com/home-assistant/core/pull/39257 and https://github.com/home-assistant/core/pull/39267 and https://github.com/home-assistant/core/pull/39268 and (doesn't need to wait to merge though since the service won't be there)

## Proposed change

Add rest and command_line to the list of reloadables

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -839,7 +839,9 @@
               "input_datetime": "Reload input date times",
               "input_select": "Reload input selects",
               "template": "Reload template entities",
-              "universal": "Reload universal media player entities"
+              "universal": "Reload universal media player entities",
+              "rest": "Reload rest entities",
+              "command_line": "Reload command_line entities"
```
Comment (zsarnett) [inline src/translations/en.json:844]: is `command_line` a unique name (with the underscore) or should it be `command line entities`

Comment (bdraco) [reply src/translations/en.json:844]: We call them Command line here (https://www.home-assistant.io/integrations/sensor.command_line/) so probably "command line entities" makes sense  👍

Comment (bdraco) [inline src/translations/en.json:844]:
```suggestion
              "command_line": "Reload command line entities"
```

**Review by zsarnett — APPROVED** [review]

**Comment by bdraco** [comment]
Retested, all looks good

**Pull request merged**